### PR TITLE
Change support for foreign kernels

### LINKED ISSUE #2763: supported kernels
**Description**
[docs/kernels.md](https://github.com/linuxkit/linuxkit/blob/master/docs/kernels.md) references various kernel variants:

> linuxkit/kernel-mainline: Mainline kernel.org kernels from the Ubuntu Mainline PPA.
> linuxkit/kernel-ubuntu: Selected Ubuntu kernels.
> linuxkit/kernel-debian: Selected Debian kernels.
> linuxkit/kernel-centos: Selected CentOS kernels.
> linuxkit/kernel-fedora: Selected Fedora kernels.

 Are they still supported?

**Steps to reproduce the issue:**

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

### PR DESCRIPTION
**Pull request by rn**
We used to periodically build linuxkit packages for kernels from other distributions. This was a manual process, so didn't happen very often. This PR abandons  this approach and instead rewrites the scripts so that users can build their own LinuxKit kernel packages from `centos`, `fedora`, `debian`, and `ubuntu` kernels.  There is also a script to re-package the mainline PPA kernels.

Once this gets merged, I'll remove the docker hub repos.

resolves #2763

bieber-ratte

### CODE REVIEW

File: scripts/kernels/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Using "foreign" kernels
+
+This directory contains a number of scripts to re-package other
+distributions kernels into a LinuxKit kernel package. The cripts
```
Comment (ijc) [inline scripts/kernels/README.md]: typo: "cripts"

File: scripts/kernels/debian.sh [hunk 2]
```diff
@@ -1,32 +1,30 @@
 #! /bin/sh
 
-REPO="linuxkit/kernel-debian"
-BASE_URL=http://mirrors.kernel.org/debian/pool/main/l/linux/
-
-TAGS=$(curl --silent -f -lSL https://registry.hub.docker.com/v1/repositories/${REPO}/tags)
-
+if [ "$#" -ne 3 ]; then
+    echo "Usage: $0 <org/repo> <first version> <second version>"
```
Comment (ijc) [inline scripts/kernels/debian.sh]: FWIW `<first version>` is really the ABI version while `<second version>` is the actual package version (assuming I've interpreted the script's behaviour correctly).

**Comment by rn** [comment]
Thanks force pushed minor fixes (type and clarification of versions in debian)

**Comment by rn** [comment]
deleted the hub repos now as well

**Pull request merged**